scripts/declarations/utils/index.test.js

```diff
@@ -145,6 +145,8 @@ async function setupRepository() {
 
   declarationUtils = new DeclarationUtils(INSTANCE_PATH, 'main');
   await declarationUtils.git.init();
+  await declarationUtils.git.addConfig('user.name', 'Open Terms Archive Testing Bot');
```

Comment:
It's not blocking but I find it a bit strange to name it as if it were a specific bot for the test.

Reply:
I have to specify a user name and email otherwise it does not work, do you have a better suggestion?

Reply:
Either `Open Terms Archive Bot` associated with `bot@opentermsarchive.org`

Or `Open Terms Archive Testing Bot` associated with `testing-bot@opentermsarchive.org`